components/Icons/ResearchCoinIcon.tsx

```diff
@@ -0,0 +1,107 @@
+import { css } from "aphrodite";
+
+type Args = {
+  onClick?: Function,
+  overrideStyle?: any,
+  height?: number,
+  width?: number,
+  version?: number,
+  color?: string,  
+}
+
+const ResearchCoinIcon = ({
```

Comment:
Previously used to be in `icons.js`. Has been extracted out so that arguments can be controlled typescript style